docs/obis-observer/types.md

```diff
@@ -3,11 +3,32 @@
 Commands use many specific data types packed to bytes.
 This document describes these types and structuring principles.
 
+# Meter profile
```

Comment:
something is wrong with hierarchy here and below

Reply:
did it